### ISSUE TITLE
Question about Pytorch Hub

### ISSUE THREAD

**Issue by YeonwooSung** [epic-kitchens/action-models#8]
Hi. I am currently trying to use the model on the PyTorch Hub. I could see how your code extracts the features and logits. However, I really have no idea how to use those extracted logits for the classification.

Is there any way to get the name of the label by using the extracted noun and verb logits?

**Comment by YeonwooSung** [comment]
Found the answer by myself. Sorry for asking silly question.
